thingsvision/core/extraction/extractors.py

```diff
@@ -134,7 +134,7 @@ def __init__(
 
     def load_model_from_source(self) -> None:
         """Load a (pretrained) neural network model from <timm>."""
-        if self.model_name in timm.list_models():
+        if self.model_name in timm.list_models() or self.model_name.split(".")[0] in timm.list_models():
```

Comment:
@a1247418 why not just `if self.model_name.split(".")[0] in timm.list_models()`?